spec/genre_spec.rb

```diff
@@ -0,0 +1,22 @@
+require 'spec_helper'
+
+describe Genre do
+  before :each do
+    @genre = Genre.new('Comedy')
+    @item = Item.new('1999/10/23')
+  end
```

Comment:
Since you were working on the `MusicAlbum` item you can create a music album instance which is more semantic, a music album having a genre
Check out this link
https://github.com/VitorGuedesMadeira/Capstone-Ruby/blob/development/spec/genre_spec.rb